overlays/ps-cv-tensorflow/Pipfile.lock

```diff
@@ -296,1147 +323,1267 @@
             "hashes": [
                 "sha256:ff26bdd583d155aa951ee3b152b3d0cffae8005dc697f72b44a8e8c2a77a8cbe"
             ],
+            "index": "pypi-org-simple",
             "version": "==1.8.1"
         },
         "tensorflow": {
             "hashes": [
-                "sha256:0701da16a3d6d34763cd9ced6467cee24c02c9abf0d1a48ba59ea5a8d0421cec",
-                "sha256:0a3b58d90fadb5bdf81a964bea73bb89019a9d1e9ac12de75375c8f65e0d7570",
-                "sha256:25e1e898bc1df521af9a8bfe0e511124379a6414083234ec67c6ab212ad12b2f",
-                "sha256:487918f4074685e213ba247387faab34933df76939134008441cb9d3e2c95cab",
-                "sha256:4b542af76d93c43e9d24dcb69888793831e434dc781c9533ee07f928fce84a15",
-                "sha256:5806d4645bce5eb415863d757b5f056364b9d1cfa2c34f711f69d46cac605eee",
-                "sha256:60d5b4fbbb7a1304d96352372fa032e861e98bb3f23aced7ce53bc475a2df97d",
-                "sha256:64cc999ae83ddd891083141d3e5d718e3d799501a1b56c544f2ca648a8396c3e",
-                "sha256:741a74278f471dc21991a6c7dc802d454d42fd39515900c6363b8c38a898fb0f",
-                "sha256:8773858cbf37aaad444b07605d29f5b2d8f7cd1ecbf1cce2777931b96884589c",
-                "sha256:9f4677e9ab7104e73710a94ff5d2ed4b335378dcd2ac7402a68c31802a680911",
-                "sha256:c588a1f34d9db51ea856aff07da9aa877c1d1d109336eee2c3bbb16dabd3f605",
-                "sha256:d9b19b5120c0b393d9e2fc72561cfa3a454ef7f1ac649d8ad0dcc98817a086a4",
-                "sha256:d9f711c5ff04333355c83eb96ca2e1db57c9663c6fa01d68b5953a040a602a3c",
-                "sha256:e129114dc529e63af9c419b5917b3407d0d26a4c8b73e114f601a175a7eb0477",
-                "sha256:e85f89bc23c62d4243fad70bac902f00a234b33da8b91e2967eeef0f4b75b1e3"
-            ],
-            "index": "pypi",
-            "version": "==2.10.0"
-        },
-        "tensorflow-estimator": {
-            "hashes": [
-                "sha256:f324ea17cd57f16e33bf188711d5077e6b2e5f5a12c328d6e01a07b23888edcd"
+                "sha256:ed47f4619dffd82d07fc0805eb2bd3039f644996f17c93d7a29ee4e455eb4632",
+                "sha256:61c3f0443c3f7e7a9d772098943bec3db0c08b76441d314e6c51edb23ca9a6eb",
+                "sha256:b44530b63a72b87adbbfbc513cc2389d893934384e9d0e3f3b7f925149f9b4f5",
+                "sha256:69cdf6c6e4b313621de5e5caf118048b7a61a315b757d0dd1e77d02f030a2e4c",
+                "sha256:3ee49c004528c003bdf431f1f8cc0ea43f668a29ea92aae6abb4ba6ce9173322",
+                "sha256:ae61123a2130ad632ee60ef784b5b90b6a0efbc955cdf87a5600f2bcc75869e0",
+                "sha256:00aa9c594c15f87ce3862d8b0c3f922453f853f3b9fdf8138ccd5f5f1bf9b2ab",
+                "sha256:ad78577b15990f475070f41fe470f0cea8f7e03b0961e1588269171a2d9e170c",
+                "sha256:6db80867a51ef1d95228b7cf8fa0a6de262100a7eb81d4718ea37ba67817b64f"
             ],
-            "markers": "python_version >= '3.7'",
-            "version": "==2.10.0"
+            "index": "pypi-org-simple",
+            "version": "==2.2.3"
```

Comment:
This version downgrade seems to introduce a few known security issues, according to https://thoth-station.ninja/search/advise/adviser-220920204912-df43cb0ddd75f56f/packages/tensorflow